### PR TITLE
updated db schema with prisma default behavior to minimize future differences

### PR DESCRIPTION
**Pull request by igoychev**

### CODE REVIEW

File: schema.prisma [hunk 1]
```diff
@@ -11,18 +11,18 @@ datasource db {
 model Person {
   id             String           @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
   firstName      String           @map("first_name") @db.VarChar(100)
-  lastName       String           @map("lastName") @db.VarChar(100)
+  lastName       String           @db.VarChar(100)
```
Comment (kachar) [inline schema.prisma:14]: Maybe this needs to be 
```suggestion
  lastName       String           @map("last_name") @db.VarChar(100)
```

File: schema.prisma [hunk 2]
```diff
@@ -37,50 +37,22 @@ model Coordinator {
...
+  /// Coordinator for this beneficiary
+  coordinatorId       String          @db.Uuid
   countryCode         String          @map("country_code") @db.Citext
   cityId              String          @map("city_id") @db.Uuid
   details             Json?
-  /// Coordinator for this beneficiary
-  coordinatorId       String          @db.Uuid
-  coordinatorRelation PersonRelation  @default(none) @map("coordinator_relation")
   created_at          DateTime        @default(now()) @map("createdAt") @db.Timestamptz(6)
   updated_at          DateTime?       @updatedAt @map("updatedAt") @db.Timestamptz(6)
```
Comment (kachar) [inline schema.prisma:51]:
```suggestion
  createdAt          DateTime        @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt          DateTime?       @updatedAt @map("updated_at") @db.Timestamptz(6)
```

File: schema.prisma [hunk 3]
```diff
@@ -90,47 +65,31 @@ model CampaignType {
...
-  description    String?       @map("description")
+  description    String?
   targetAmount   Decimal?      @map("target_amount") @db.Decimal
-  currency       String?       @default("BGN")
+  currency       String?       @default("BGN") @db.VarChar(3)
   startDate      DateTime?     @map("start_date") @db.Timestamptz(6)
   endDate        DateTime?     @map("end_date") @db.Timestamptz(6)
   createdAt      DateTime      @default(now()) @map("created_at") @db.Timestamptz(6)
-  updatedAt      DateTime?     @updatedAt @map("updated_at") @db.Timestamptz(6)
+  updatedAt      DateTime?     @map("updated_at") @db.Timestamptz(6)
```
Comment (kachar) [inline schema.prisma:92]: It's better to mark the field with `@updatedAt` annotation add will tell Prisma to automatically update the value on update queries.

Read more at https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/schema-incompatibilities-postgres#updatedat-isnt-represented-in-database

Comment (igoychev) [reply schema.prisma]: fully agree - now added @updatedAt annotation to all updatedAt fields in schema.prisma

**Pull request merged**